lib/main.dart

```diff
@@ -62,6 +64,15 @@ class _TMailAppState extends State<TMailApp> {
     if (PlatformInfo.isMobile) {
       _deepLinksManager = getBinding<DeepLinksManager>();
       _deepLinksManager?.registerDeepLinkStreamListener();
+    } else if (AppConfig.isCozyIntegrationEnabled) {
+      final cozyConfig = CozyConfig();
+  
+      cozyConfig.manager.injectCozyScript().then((_) async {
+        final isInsideCozy = await cozyConfig.isInsideCozy;
+        if (!isInsideCozy) return;
+
+        await cozyConfig.manager.initialize();
+      });
```

Comment:
Do we need to make sure that `injectCozyScript ` is loaded before entering the app?